incubator/binding-mqtt-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/mqtt/kafka/schema/mqtt.kafka.schema.patch.json

```diff
@@ -59,11 +59,22 @@
                                         "title": "Kafka Retained Topic",
                                         "type": "string",
                                         "default": "mqtt_retained"
```

Comment:
Let's also remove `default` values from the schema as the properties are now `required`.